vars/withPipeline.groovy

```diff
@@ -130,6 +130,11 @@ def call(type, String product, String component, Closure body) {
             component: component,
           )
 
+          sectionSyncBranchesWithMaster(
+            branchestoSync: pipelineConfig.branchesToSyncWithMaster,
+            product: product
+          )
```

Comment:
this is just for testing I assume?

Reply:
Yep just testing if (!branchesToSync.isEmpty()) to see if it disables the sync if it is set to empty or does it default to ['ithc', 'demo', 'perftest']